tchannel-core/src/main/java/com/uber/tchannel/channels/ChannelManager.java

```diff
@@ -41,47 +39,102 @@
  * entry point for shutting down all active Channels.
  */
 public class ChannelManager {
```

Comment:
If all of these methods are returning Connections, how about renaming this ConnectionManager instead? Seems misleading otherwise.